client/src/main/java/com/alibaba/nacos/client/logging/log4j2/Log4J2NacosLogging.java

```diff
@@ -45,10 +46,13 @@
     
     private static final String NACOS_LOGGER_PREFIX = "com.alibaba.nacos";
     
-    private final String location = getLocation(NACOS_LOG4J2_LOCATION);
-    
     @Override
     public void loadConfiguration() {
+        final String location = getLocation(NACOS_LOG4J2_LOCATION);
+        if (StringUtils.isBlank(location)) {
```

Comment:
Only add this judgement is ok.

otherwise the location maybe read multiple time.

Reply:
done.

I don't know if our project  will squash commits togethor ?
I amend and  fource push  again.